From 7147b3e8280d2ae245b28e9f6ede1d9210af79ef Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jcline@redhat.com>
Date: Tue, 5 Jun 2018 23:00:42 -0400
Subject: [PATCH] Allow urllib3-1.23

Signed-off-by: Jeremy Cline <jcline@redhat.com>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index ed4892d4..e113510c 100755
--- a/setup.py
+++ b/setup.py
@@ -44,7 +44,7 @@ packages = ['requests']
 requires = [
     'chardet>=3.0.2,<3.1.0',
     'idna>=2.5,<2.7',
-    'urllib3>=1.21.1,<1.23',
+    'urllib3>=1.21.1,<1.24',

 ]
 test_requirements = ['pytest-httpbin==0.0.7', 'pytest-cov', 'pytest-mock', 'pytest-xdist', 'PySocks>=1.5.6, !=1.5.7', 'pytest>=2.8.0']

-- 
2.17.1

From 015347ed9622f526f045b2199a82ee57269c1260 Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jcline@redhat.com>
Date: Thu, 7 Jun 2018 11:58:46 -0400
Subject: [PATCH] Don't warn about compatibility with urllib3 v1.23

Requests allows v1.23 in #4669, but the compatibility check at import
time still looks for 22. Bump the version to 23.

Signed-off-by: Jeremy Cline <jcline@redhat.com>
---
 requests/__init__.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/requests/__init__.py b/requests/__init__.py
index 6fa855df..03decf8c 100644
--- a/requests/__init__.py
+++ b/requests/__init__.py
@@ -60,7 +60,7 @@ def check_compatibility(urllib3_version, chardet_version):
     # urllib3 >= 1.21.1, <= 1.22
     assert major == 1
     assert minor >= 21
-    assert minor <= 22
+    assert minor <= 23
 
     # Check chardet for compatibility.
     major, minor, patch = chardet_version.split('.')[:3]
-- 
2.17.1

