From e74bf477e53f9089e6d763d676333c9d84ab71e0 Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Wed, 5 Nov 2014 09:38:42 -0500
Subject: [PATCH] Remove nested budnling dep.

---
 requests/compat.py | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/requests/compat.py b/requests/compat.py
index bdf10d6..29cce93 100644
--- a/requests/compat.py
+++ b/requests/compat.py
@@ -89,9 +89,13 @@ if is_py2:
     import cookielib
     from Cookie import Morsel
     from StringIO import StringIO
-    from .packages.urllib3.packages.ordered_dict import OrderedDict
     from httplib import IncompleteRead
 
+    try:
+        from collections import OrderedDict # py2.7
+    except:
+        from ordereddict import OrderedDict # py2.6 and lower (el6, etc.)
+
     builtin_str = str
     bytes = str
     str = unicode
-- 
1.9.3

