From 2cecc4bc828fc1754cc26f279ec3ff5cd65cb8f7 Mon Sep 17 00:00:00 2001
From: Ralph Bean <rbean@redhat.com>
Date: Thu, 31 Jan 2013 21:38:58 -0500
Subject: [PATCH] cookie handling

---
 requests/models.py | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/requests/models.py b/requests/models.py
index 2193c6e..0a12237 100644
--- a/requests/models.py
+++ b/requests/models.py
@@ -17,7 +17,7 @@ from .structures import CaseInsensitiveDict
 from .status_codes import codes
 
 from .auth import HTTPBasicAuth, HTTPProxyAuth
-from .cookies import cookiejar_from_dict, extract_cookies_to_jar, get_cookie_header
+from .cookies import cookiejar_from_dict, extract_cookies_to_jar, get_cookie_header, RequestsCookieJar
 from .packages.urllib3.exceptions import MaxRetryError, LocationParseError
 from .packages.urllib3.exceptions import TimeoutError
 from .packages.urllib3.exceptions import SSLError as _SSLError
@@ -207,7 +207,8 @@ class Request(object):
                     extract_cookies_to_jar(self.cookies, self, resp)
 
                 # Save cookies in Response.
-                response.cookies = self.cookies
+                response.cookies = RequestsCookieJar()
+                extract_cookies_to_jar(response.cookies, self, resp)
 
                 # Save cookies in Session.
                 for cookie in self.cookies:
-- 
1.8.1

