From e44b99214a953fcc68227a7182f2fecd55c58267 Mon Sep 17 00:00:00 2001
From: Jeremy Cline <jeremy@jcline.org>
Date: Wed, 17 May 2017 11:57:23 -0400
Subject: [PATCH] python requests urllib3 at 1.21.1

Signed-off-by: Jeremy Cline <jeremy@jcline.org>
---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6ad0bc3..50ae1a1 100755
--- a/setup.py
+++ b/setup.py
@@ -48,7 +48,7 @@ packages = [
     'requests.packages.urllib3.contrib._securetransport',
 ]
 
-requires = []
+requires = ['urllib3==1.21.1']
 test_requirements = ['pytest>=2.8.0', 'pytest-httpbin==0.0.7', 'pytest-cov', 'pytest-mock']
 
 with open('requests/__init__.py', 'r') as fd:
-- 
2.9.4

